### ISSUE TITLE
A11y_SQLAzureDataStudio_Notebook_Create new notebook_AI4D: The control type of editor is set as "Custom" 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9367]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#AI4D;#MAS1.3.1;#MAS4.1.2;#MAS4.2.1;

Environment Details:
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

Additional Details:AMAS References:MAS4.1.2, MAS4.2.1, MAS1.3.1
Tools: accessibility insights for desktop

Repro Steps:
1. Launch Azure Data Studio.
2. Click File then New Notebook.
3. Launch accessibility insights for desktop
4. Hover over the editors.

**Actual**
The control set for editors is set as custom both code and text editor.

**Expected:**
The control should be set as editor and announced by a screen reader.

User Impact:
Screen reader users will not know that have reached the editor.

**Attachment link for Reference**
![image](https://user-images.githubusercontent.com/24580570/75446424-694aec00-598d-11ea-8c13-fa17733642a9.png)

**Comment by chlafreniere** [comment]
@fsteffi this control encompasses both an editor and the editor's outputs. I'm worried that us saying that the control type as an editor would be confusing, since it's not strictly true. Any thoughts?

**Comment by Charles-Gagnon** [comment]
@chlafreniere FYI I'm going to bring this one to the UX office hours to see what they say - since yeah there isn't a clear indication of what role this should actually be (editor isn't even an actual aria role)

**Comment by chlafreniere** [comment]
Thanks, @Charles-Gagnon !!!

**Comment by v-jagansai** [comment]
#closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
